eks-cluster-setup/cluster.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: eksctl.io/v1alpha5
+availabilityZones:
```

Comment:
I'd probably add in the README that this cluster definition is going to be deployed in certain zones, with certain configurations like the instance type, max and min sizes as well as other important info.